homeassistant/components/openrgb/strings.json

```
@@ -2,17 +2,16 @@
   "config": {
     "step": {
       "user": {
-        "title": "Set up OpenRGB connection",
-        "description": "Enter the host and port of your OpenRGB SDK server",
+        "description": "Set up your OpenRGB SDK server to allow control from within Home Assistant.",
         "data": {
           "name": "[%key:common::config_flow::data::name%]",
           "host": "[%key:common::config_flow::data::host%]",
           "port": "[%key:common::config_flow::data::port%]"
         },
         "data_description": {
-          "name": "A name for this OpenRGB entry, like the name of the computer running the SDK server.",
-          "host": "The hostname or IP address of the computer running the OpenRGB SDK server.",
-          "port": "The port on which the OpenRGB SDK server is accessible."
+          "name": "A name for this integration entry, like the name of the computer running the OpenRGB SDK server.",
+          "host": "The IP address or hostname of the computer running the OpenRGB SDK server.",
+          "port": "The port number on which the OpenRGB SDK server is running on."
```

New grammar issue now with "on" duplicated.

```suggestion
          "port": "The port number that the OpenRGB SDK server is running on."
```

---

Already suggested some changes above. What would you prefer? https://github.com/home-assistant/core/pull/153373#discussion_r2401458611

---

@tr4nt0r We can use your suggestions as well. As long as the grammar is correct that is just fine with me.

I would have to learn more about the technical details of the server setup etc. to know what to emphasize by using one or  the other wording.

---

Sorry for the grammar issue (fixed!).

BTW I have carefully analyzed @tr4nt0r's suggestion. I changed my translations a bit based on it, but after taking inspiration from several other integrations I decided to keep it how it is right now, as I think it follows existing patterns more closely and is better explicative.

But of course, I will change if need be.